test/stories/CodeSandbox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { DEFAULT_REACT_CODESANDBOX, DEFAULT_VUE3_CODESANDBOX, DEFAULT_VUE_CODESANDBOX } from "../../dist";
```

Comment:
Does this mean that the tests should run after the main package's build?
Then, shouldn't it be specified on the test script?
or, can it be imported from `src` instead?